client/app/nonComp/components/ReportPage/SaveLimitReachedModal.jsx

```diff
@@ -0,0 +1,44 @@
+
+import React from 'react';
+import Modal from 'app/components/Modal';
+import COPY from 'app/../COPY';
+import PropTypes from 'prop-types';
+
+export const SaveLimitReachedModal = (props) => {
```

Comment:
We could do the props like this and remove a line

```suggestion
export const SaveLimitReachedModal = ({ setShowLimitModal }) => {
```